docs/source/concept_models/vivarium_swissre_colorectalcancer/concept_model.rst

```diff
@@ -470,29 +490,37 @@ Relevant formulas
      - scale up to 60% in 2030
      - [de-Moor-et-al-2018]_
      - We applied 20-year lag on US coverage
-   * - Screening sensitivity
-     - 
-     - 
+   * - FOBT sensitivity
```

Comment:
It appears that these numbers are listed in two places in this document.  Please delete one---that will make it impossible for them to get out-of-sync